Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM debian:9
+
+RUN mkdir /var/run/sshd
+RUN apt-get update
+RUN apt-get install -y openssh-server sudo
+
+# Set a locale with UTF-8 support
+# It is needed by python to run bitbake
+RUN apt-get update && apt-get install -y locales && rm -rf /var/lib/apt/lists/* \
+  && localedef -i en_US -c -f UTF-8 -A /usr/share/locale/locale.alias en_US.UTF-8
+ENV LANG en_US.utf8
+
+RUN useradd vagrant --create-home --user-group --groups sudo
+RUN echo 'vagrant:vagrant' | chpasswd
+
+RUN echo 'vagrant ALL=(ALL) NOPASSWD:SETENV: ALL' > /etc/sudoers.d/vagrant
+
+EXPOSE 22
+CMD ["/usr/sbin/sshd", "-D"]
```

Comment:
Shouldn't you do that via systemd instead? Especially the next time you start a docker container you want it running too don't you? Or if you run everything via the docker command you don't need sshd there, getting the artifacts could work by bindmounting in a directory from the outside.